test/sycl-kernel-arg-annotation.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Please move the test to test/extensions/INTEL/SPV_INTEL_fpga_argument_interfaces directory

Reply:
Done